src/libcrun/utils.c

```diff
@@ -2507,3 +2507,17 @@ get_overflow_gid (void)
     }
   return gid;
 }
+
+void
+consume_trailing_slashes (char *path)
```

Comment:
I didn't add it last time, but this could use the `inline` hint, too. Maybe compiler will be kind to us.